spec/models/offspring_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'rails_helper'
+
+RSpec.describe Offspring, type: :model do
+  describe "validating attribute first_name " do
+    it "should be required" do
+      blank = Factory.build(:offspring, first_name: "")
+      blank.should_not be_valid
+      blank.errors[:first_name].should include("can't be blank")
```

Comment:
FactoryGirl... Why are you using Factory?
It should shout when using guard that there is an error on Code